operations-stdlib/src/iosMain/kotlin/Format.kt

```diff
@@ -0,0 +1,58 @@
+import kotlinx.cinterop.cstr
+import operation.StandardLogicOperation
+import platform.Foundation.NSString
+import platform.Foundation.stringWithFormat
+import utils.asList
+import utils.secondOrNull
+
+actual object Format : StandardLogicOperation {
+    actual override fun evaluateLogic(expression: Any?, data: Any?): Any? {
+        return with(expression.asList) {
+            val format = firstOrNull().toString()
+            val args = secondOrNull().asList
+
+            runCatching { format.formatString(args) }
+                .fold(
+                    onSuccess = { it },
+                    onFailure = { null }
+                )
+        }
+    }
+
+    private fun String.formatString(args: List<Any?>): String {
+        val regEx = "%[\\d|.]*[sdf]|[%]".toRegex()
+        val formats = regEx.getFormats(this)
+        val rawStrings = this.split(regEx).toMutableList()
```

Comment:
I'm also not a huge fan of using MutableList here - it might be error prone
Maybe change it like this?
```
    private fun String.formatString(args: List<Any?>): String {
        val formats = REGEX.getFormats(this)
        if (args.isEmpty() || formats.isEmpty()) {
            return this
        }

        val rawStrings = this.split(REGEX)
        return args.formatText(rawStrings.toMutableList(), formats)
    }
```
The same for format, but sequence complicates things a bit (forces to first convert to list and then mutable list)

Reply:
using mutable list as param one makes no sense. 
Whole point is in poping first element of the list is to easily join the rest into string without checking indexes etc.
Do you think that cutting from raw strings count of args is less error prone?